stage1/enter/enter.c

```diff
@@ -27,6 +27,12 @@
 #include <sys/wait.h>
 #include <unistd.h>
 
+#ifdef USE_NR_SETNS_MACRO_INSTEAD
```

Comment:
Would it be better to use
```
#ifndef setns
#include <linux/unistd.h>
static int setns(int fd, int nstype) {
  return syscall(__NR_setns, fd, nstype);
}
#endif
```
instead of declaring `USE_NR_SETNS_MACRO_INSTEAD` in `configure.ac`.

Then, no need to modify the call sites of `setns()`